src/sfizz/Range.h

```diff
@@ -32,6 +32,12 @@ class Range {
     {
 
     }
+
+    constexpr Range(const Range<Type, !Checked>& other)
+        : Range(other.getStart(), other.getEnd())
+    {
+    }
+
     constexpr Type getStart() const noexcept { return _start; }
```

Comment:
In the rest of the file all the `if (Checked) ...` could use the `IF_CONSTEXPR` macro for C++17, I think we have it.